owncloudApp/src/main/java/com/owncloud/android/presentation/files/filelist/MainFileListFragment.kt

```diff
@@ -593,6 +598,24 @@ class MainFileListFragment : Fragment(),
                 }
             }
         }
+
+        /* TransfersViewModel observables */
+        // Observe the transfers
+        collectLatestLifecycleFlow(transfersViewModel.transfersWithSpaceStateFlow) { transfers ->
```

Comment:
If you take a look at the container method, it is `subscribeToViewModels()`, and it is where we add the observers of the different ViewModels in fragments usually. Moving each of the observers out of this method as new methods could be an option, but since they are used just once and `subscribeToViewModels()` has a simple structure, I think we can keep it like that

Reply:
A method should do one thing, this method do a lot of thing. Actually, you had to add some comments to clarify what is done in that part of the code. Don't you think it would be better to isolate that action in a method? You would save on comments, and the code would be more readable, as you wouldn't have to read such a long method.